llama_index/embeddings/llm_rails.py

```diff
@@ -0,0 +1,67 @@
+import logging
+from typing import Any, List
+
+import requests
+from llama_index.embeddings.base import BaseEmbedding
+
+logger = logging.getLogger(__name__)
+
+
+class LLMRailsEmbeddings(BaseEmbedding):
+    """LLMRails embedding models.
+
+    This class provides an interface to generate embeddings using a model deployed
+    in an LLMRails cluster. It requires a model_id of the model deployed in the cluster and api key you can obtain 
+    from https://console.llmrails.com/api-keys.
+
+    """  
+
+    model_id: str
+    api_key: str
+
+    @classmethod
+    def class_name(self) -> str:
+        return "LLMRailsEmbeddings"
+
+    def __init__(
+        self,
+        api_key: str,
+        model_id: str = 'embedding-english-v1', # or embedding-multi-v1
+        **kwargs: Any,
+    ):
+        super().__init__(model_id=model_id, api_key=api_key, **kwargs)
+
+    def _get_embedding(self, text: str) -> List[float]:
+        """
+        Generate an embedding for a single query text.
+
+        Args:
+            text (str): The query text to generate an embedding for.
+
+        Returns:
+            List[float]: The embedding for the input query text.
+        """
+        try:
+            response = requests.post('https://api.llmrails.com/v1/embeddings',
```

Comment:
instead of using requests, we could use `httpx` to provide proper async functions for `_aget_query_embedding` and `_aget_text_embedding`

Reply:
Yep I also thought about using aio http I am gonna update it now

Reply:
@logan-markewich Check please